pkg/processor/util/updater/updater.go

```diff
@@ -0,0 +1,214 @@
+/*
```

Comment:
This file has potential to grow, please preemptively split into partitionchanger.go, partitionadder.go, partitionremover.go, etc

Reply:
It's only 229LOC currently, I agree that if it'll grow we can split it but I think it won't grow that much since the functionality of *how* to remove/add/... is in the triggers.

I'd rather keep it in one file for now.